cargo-shuttle/src/lib.rs

```diff
@@ -1079,7 +1081,13 @@ impl Shuttle {
                     shuttle_common::log::Backend::Runtime(service_name_clone.clone()),
                     line,
                 );
-                println!("{log_item}");
+                if !quite_mode {
+                    println!("{log_item}");
+                } else {
+                    if log_item.line.contains("ERROR") {
+                        println!("{log_item}");
+                    }
+                }
```

Comment:
We can't filter the runtime's lines like this. The goal is to supress shuttle's prints in cargo-shuttle and runtime. No lines from the user's program should be silenced.

Reply:
Okay.. Thanks : ), will try to do that.